core/src/main/java/cz/xtf/core/openshift/OpenShift.java

```diff
@@ -828,6 +834,8 @@ public BuildConfig updateBuildConfig(BuildConfig buildConfig) {
      *
      * @param name name of buildConfig
      * @param envVars environment variables
+     *
+     * @return returns build config
```

Comment:
`@returns` are incosistent, `ProjectRequest` / `deployment config` / `returns build config`

Reply:
I went through the `@returns` and fixed those.